include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -9569,20 +9569,26 @@ def err_bounds_type_annotation_lost_checking : Error<
 
   def err_initializer_expected_for_ptr : Error<
     "automatic variable %0 with _Ptr type must have initializer">;
-
-  def err_initializer_expected_for_integer_with_bounds_expr : Error<
-    "integer variable %0 with a bounds expression must have an initializer">;
-
-  def err_initializer_expected_for_unchecked_pointer_in_checked_scope_with_bounds_expr : Error<
-    "unchecked pointer variable %0 in a checked scope with a bounds expression must have an initializer">;
-
-  def err_initializer_expected_for_array : Error<
-    "array variable %0 with elements containing checked pointers must have an initializer">;
-	
-  def err_initializer_expected_for_record_with_checked_value : Error<
+
+  def err_initializer_not_null_terminated_for_nt_checked : Error<
+      "null terminator expected in NT_CHECKED array initializer">;
```

Comment:
Some of these diagnostics look unchanged.  Please avoid unnecessary white space changes.   Formatting changes should be done separately.